apps/patient/src/graphql/queries.ts

```diff
@@ -22,6 +25,8 @@ export const GET_ORDER = gql`
         type
         state
         trackingNumber
+        minutesUntilReady
+        createdAt
```

Comment:
Is `minutesUntilReady` actually relative to `createdAt`? It looks to me from reading the code that `minutesUntilReady` is relative to that being set whereas `createdAt` happens when the order is sent?